lib/recipes/ap-physics/ap_physics_recipe.rb

```diff
@@ -135,14 +135,3 @@
   BakeFolio.v1(book: book)
   BakeLinks.v1(book: book)
 end
-
-opts = Slop.parse do |slop|
-  slop.string '--input', 'Assembled XHTML input file', required: true
-  slop.string '--output', 'Baked XHTML output file', required: true
-end
-
-puts Kitchen::Oven.bake(
-  input_file: opts[:input],
-  recipes: [AP_PHYSICS_RECIPE],
-  output_file: opts[:output]
-)
```

Comment:
@KarinaMendez2 Are these lines removed accidentally?

Reply:
no, I intentionally did because this is baking twice, these lines should only be called in the ap_physics bake file, I forgot to remove them in that PR :(